modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -16,21 +16,8 @@ const trimEmptyContents = (editor: Editor, html: string): string => {
   return html.replace(emptyRegExp, '');
 };
 
-const getPlainTextContent = (editor: Editor, body: HTMLElement) => {
-  const doc = editor.getDoc();
-  const dos = SugarShadowDom.getRootNode(SugarElement.fromDom(editor.getBody()));
-
-  const offscreenDiv = SugarElement.fromTag('div', doc);
-  Attribute.set(offscreenDiv, 'data-mce-bogus', 'all');
-  Css.setAll(offscreenDiv, {
-    position: 'fixed',
-    left: '-9999999px',
-    top: '0'
-  });
-  Html.set(offscreenDiv, body.innerHTML);
-
-  // Cleanup bogus elements
-  const bogusElements = SelectorFilter.descendants(offscreenDiv, '[data-mce-bogus]');
+export const cleanupBogusElements = (parent: SugarElement<Node>): void => {
```

Comment:
Should those cleanup helpers be in a separate module though? as it’s a bit messy having one here and then `GetSelectionContentImpl` import from `GetContentImpl`. It'd also help in the long run since we should merge the 2 get text bits of logic as well (though that's really outside of the scope here).